dusty/daemon.py

```diff
@@ -34,6 +34,9 @@ def _send_warnings_to_client(connection, suppress_warnings):
     if daemon_warnings.has_warnings and not suppress_warnings:
         connection.sendall("{}\n".format(daemon_warnings.pretty()))
 
+def _get_payload_function_data(payload):
+    return payload['fn'], payload['args'], payload['kwargs']
```

Comment:
That should not be needed anymore because I am just passing in a strait dict to the pickle function.  Recasting what should already be a dict to a dict seems silly